xontrib/voxapi.py

```diff
@@ -7,16 +16,45 @@
 
 from xonsh.platform import ON_POSIX, ON_WINDOWS, scandir
 
+# This is because builtins aren't globally created during testing.
+# FIXME: Is there a better way?
```

Comment:
I am not sure that there is a better way right now, other than having a xontrib load event that does the importing.  But for that we'd need a xontrib load event :)